src/Microsoft.DotNet.Interactive/Kernel.cs

```diff
@@ -189,6 +189,14 @@ private bool TryPreprocessLanguageServiceCommand(LanguageServiceCommand command,
             }
 
             commands = postProcessCommands;
+            if (commands.Count == 1 &&
+                commands[0] is LanguageServiceCommand languageCommand &&
```

Comment:
The intent here isn't clear. We should extract a common way here and elsewhere to identify commands to be fast-tracked.

Reply:
This isn't about fast-tracked commands, it's just a place we found where language service commands were unnecessarily being split.  The normal submission parser does something similar, we just forgot to do it here.

Reply:
It's still worth consolidating this logic. It's not really clear what the "why" is here. Extracting a named method could help.

Reply:
I had to pull this change out, since it caused problems with how we compute language service commands and offsets.